internal/service/contact.go

```diff
@@ -40,25 +40,18 @@ func (c contactService) GetByUserID(userID uint) ([]model.Contact, error) {
 	return c.contactRepository.GetByUserID(userID)
 }
 
-func (c contactService) DeleteByID(id, userID uint) error {
-	contact, err := c.contactRepository.GetByID(id)
-	if err != nil {
-		return err
-	}
-
-	if contact.UserID != userID {
-		return errors.New("unauthorized to delete contact")
-	}
-
-	return c.contactRepository.DeleteByID(id)
+// Zwraca error jeżeli nie udało się nic usunąć (bo np. nie było co usunąć lub "chcemy usunąć nie nasz kontakt")
```

Comment:
Comments should be in english and there should be whitespace between `//` and message